src/packaging/bin/spreaper

```diff
@@ -638,11 +638,11 @@ class ReaperCLI(object):
         
         if args.node:
             print ("# Deleting snapshot '{1}' on cluster '{0}' and node '{2}'" 
```

Comment:
Please check the ordering of arguments for this one. I think you've reversed cluster name and snapshot name.